test/integration/features/cmd-config.feature

```diff
@@ -174,10 +174,17 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
-    | iso-url         | file:///home/me/Downloads/my_handmade_b2d.iso                                                    | file:///home/me/Downloads/my_handmade_b2d.iso                                                   |
     | iso-url         | b2d                                                                                              | b2d                                                                                             |
     | iso-url         | centos                                                                                           | centos                                                                                          |
-
+       
+  @quick
+  Scenario: Setting iso-url key to a non exsisting path
```

Comment:
typo

Reply:
@agajdosi Fixed it! Sorry for the delay..